mltsp/datasets/andrzejak.py

```diff
@@ -0,0 +1,149 @@
+import glob
+import logging
+import os
+import tarfile
+from zipfile import ZipFile
+
+import numpy as np
+import pandas as pd
+from sklearn.datasets.base import Bunch
+from sklearn.externals import joblib
+
+import mltsp.cfg
+import mltsp.util
+
+try:
+    from urllib.request import urlopen
+except:
+    from urllib2 import urlopen
+
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_DIR = os.path.join(mltsp.cfg.DATA_PATH, "andrzejak")
+BASE_URL = "http://epileptologie-bonn.de/cms/upload/workgroup/lehnertz/"
+ZIP_FILES = ["Z.zip", "O.zip", "N.zip", "F.zip", "S.zip"]
+CACHE_NAME = "andrzejak.pkl"
+ARCHIVE_NAME = "andrzejak.tar.gz"
+T_MAX = 23.6
+
+
+def download_andrzejak(data_dir):
```

Comment:
What would those functions look like? Each is just a call to `urlopen` or `Zipfile.extractall`, respectively.